program/src/state/stake.rs

```diff
@@ -0,0 +1,51 @@
+use bytemuck::{Pod, Zeroable};
+use shank::ShankAccount;
+use solana_program::{clock::UnixTimestamp, pubkey::Pubkey};
+use spl_discriminator::SplDiscriminate;
+
+/// Data for an amount of tokens staked with a validator
+#[repr(C)]
+#[derive(Clone, Copy, Pod, ShankAccount, SplDiscriminate, Zeroable)]
+#[discriminator_hash_input("stake::account")]
```

Comment:
Ahh my bad, I forgot about this discriminator. What should we name this one, since we went with `stake::config` in the other? `stake::stake`?

Reply:
Is `stake::account` confusing? So we would have a `config` and `account` types.

Reply:
We could rename the `Stake` struct to `Account` as well.

Reply:
In my opinion the struct name and the discriminator should match, and the pattern should be the same for both account types. So yeah I'd say `stake::account` is confusing with the current pattern.

Reply:
Agree, should we name the struct `Account` or should we change the discriminator to `stake::stake`. I have a slight preference for having the struct named `Account`, but happy either way.

Reply:
What about just doing `stake::state::config` and `stake::state::stake`? We could also do `stake::account::config` and `stake::account::stake`. What do you think?

Reply:
Oh, nice idea! I like the `stake::state::*` pattern.